server/routes/ad.ts

```diff
@@ -589,9 +596,13 @@ export default class AdService {
 
       //Given each detector from previous result, get aggregation to power list
       const allDetectorIds = Object.keys(allDetectorsMap);
+      let requestParams = {
+        resultIndex: CUSTOM_AD_RESULT_INDEX_PREFIX + '*',
```

Comment:
can you explain how this includes results from custom result indices and non-custom results indices?

Reply:
After discussion, understood that the backend will search on the system indices by default, + results from any specified custom result index. So by specifying all custom result indices, all results (from both system indices + any custom results indices) will be returned here.

Can you add a comment in the code about this @ylwu-amzn ?

Reply:
Sure, done.